Makefile

```diff
@@ -419,14 +421,11 @@ generate-mocks: install-mockery
 echo-version:
 	@echo $(GIT_CLOSEST_TAG)
 
-PROTOC := protoc
+PROTOC := docker run --rm -v${PWD}:${PWD} -w${PWD} ${JAEGER_DOCKER_PROTOBUF} --proto_path=${PWD}
 PROTO_INCLUDES := \
-	-I model/proto \
-	-I idl/proto \
-	-I vendor/github.com/grpc-ecosystem/grpc-gateway \
-	-I vendor/github.com/gogo/googleapis \
-	-I vendor/github.com/gogo/protobuf/protobuf \
-	-I vendor/github.com/gogo/protobuf
+	-Imodel/proto \
+	-Iidl/proto \
+	-I/usr/include/github.com/gogo/protobuf
```

Comment:
I'm curious, why is this last import needed? Isn't it already present in the Docker image?

Reply:
The Docker image has `-I/usr/include` but I think we needed this relative path as well